x-pack/plugins/security/public/views/management/edit_role/components/edit_role_page.tsx

```diff
@@ -292,21 +291,21 @@ export class EditRolePage extends Component<Props, State> {
           this.backToRoleList();
         })
         .catch((error: any) => {
-          notifier.error(get(error, 'data.message'));
+          toastNotifications.addDanger(error.data.message || 'Error saving role');
```

Comment:
I think we still want the `get(error, 'data.message')` as it's possible for us to have an error without the `data` property and this causes the browser to log an error to the console

Reply:
good call, I'm not sure why I deviated from what was there